src/projectscene/view/clipsview/trackslistclipsmodel.cpp

```diff
@@ -106,7 +110,7 @@ QVariant TracksListClipsModel::data(const QModelIndex& index, int role) const
     case TrackIdRole:
         return QVariant::fromValue(track.id);
     case IsDataSelectedRole: {
-        return muse::contains(m_dataSelectedTracks, track.id);
+        return muse::contains(m_selectedTracks, track.id) && selectionController()->timeSelectionIsNotEmpty();
```

Comment:
That makes a lot of sense now.
I'm wondering more and more if `m_selectedTracks` couldn't disappear in favor of calls to `selectionController()->selectedTracks()`, to avoid this app state being maintained in two places. Or does something speak against it?